### PR TITLE
Enable Airskin node for tuw-robotino2

### PR DESCRIPTION
**Pull request by bajo**
Rotary and arm - controllers listen to airskin bumper output for pause command

### CODE REVIEW

File: robotino_bringup/robots/tuw-robotino2/launch/tuw-robotino2.launch [hunk 1]
```diff
@@ -36,14 +36,18 @@
     <arg name="scan_topic" value="/scan" />
   </include>
 
+  <include file="$(find airskin)/launch/startup.launch">
```
Comment (NadiaHG) [inline robotino_bringup/robots/tuw-robotino2/launch/tuw-robotino2.launch:39]: It requires a dependency to airskin package

File: robotino_bringup/robots/tuw-robotino2/launch/tuw-robotino2.launch [hunk 2]
```diff
@@ -36,14 +36,18 @@
...
+
   <node name="robotino_odometry_node" pkg="robotino_driver" type="robotino_odometry_node" output="screen">
     <param name="hostname" value="$(arg hostname)" />
   </node>
 
   <group ns="rotatory_controller">
       <remap from="robot_description" to="/robot_description" />
       <node name="robotino_arm_base_control_node" pkg="robotino_arm_base_control" type="robotino_arm_base_control_node" output="screen">
-        <remap from="pause" to="/bumper" />
+        <remap from="pause" to="/airskin/arm_bumper" />
```
Comment (NadiaHG) [inline robotino_bringup/robots/tuw-robotino2/launch/tuw-robotino2.launch:50]: already tested on the robot?

**Comment by ipa-nhg** [comment]
@bajo @mzillich Is there a model of the skin? it is possible to use a gazebo bumper collision to simulate the skin behaviour.

**Comment by mzillich** [comment]
There is no 3D model. But we can certainly get a mesh model file. Which format? STL?

**Comment by ipa-nhg** [comment]
STL is perfect!

File: robotino_bringup/robots/tuw-robotino2/launch/tuw-robotino2.launch (hunk 1 above)
Comment (bajo) [inline robotino_bringup/robots/tuw-robotino2/launch/tuw-robotino2.launch:39]: @ipa-nhg Ok, I will add this.

File: robotino_bringup/robots/tuw-robotino2/launch/tuw-robotino2.launch (hunk 2 above)
Comment (bajo) [inline robotino_bringup/robots/tuw-robotino2/launch/tuw-robotino2.launch:50]: @ipa-nhg Yes, I committed it after I changed and tested it on our robot.

**Comment by mzillich** [comment]
STL files are here [https://owncloud.tuwien.ac.at/index.php/s/VwA4Ncu5pu0MP4t]
They are full detail and large, so will have to be downsampled somehow.

**Comment by bajo** [comment]
@ipa-nhg @mzillich I fear that the Travis check will not be able to build this package as long as we depend on airskin which is not public right now. Is there anything we can do here ? Or can we at least use the buildbot internally to make sure everything still builds?

**Comment by ipa-nhg** [comment]
I have two solutions
1. skip travis and only use Buildbot to test this package, but it will also requires some extra work on Buildbot. 
2.  create a new public repository, squirrel_substitute, where we create packages with the same names as the private ones but without the source code. Just to ensure that  a external person can compile and use our simulation cloning only our public repos (squirrel_common, squirrel_robotino and squirrel_substitute) , also can be useful for the project members, it will not be necessary any more  download  and compiles all the packages to test something.

@mzillich  @bajo What do you think? I prefect the second option.

**Comment by bajo** [comment]
@ipa-nhg @mzillich I would also go with the second option. Especially in this case as we only have a run dependency.

**Comment by mzillich** [comment]
Can we not make squirrel_driver public? The substitution layer seems to make things more complicated. And I am already frightened by git as it is :)

**Comment by ipa-nhg** [comment]
We can make squirrel_driver public (if KCL is agree). But we have still problems with the arm and hand drivers. I didn't test the squirrel_robotino repository alone, but you should not be able to start the simulation because of the arm and hand descriptions, at the moment these packages are just ignored.

An example of a substitute repository is: https://github.com/ipa320/cob_substitute.  I will review squirrel_robotino next week , and check what it is at this moment working with just the public repositories --> New issue https://github.com/squirrel-project/squirrel_robotino/issues/124

**Comment by ipa-nhg** [comment]
Please take a look on https://github.com/squirrel-project/squirrel_substitute and the following pull request: https://github.com/squirrel-project/squirrel_robotino/pull/126 , travis can compile all the squirrel_robotino packages with the rights dependencies.

What do you think?

**Pull request merged**